### DISCUSSION TITLE
Get Available Markets API missing?

### DISCUSSION THREAD

**Discussion by esskar** [JohnnyCrazy/SpotifyAPI-NET#786] [Q&A] [answered]
Hi,

thanks for this create library, it helps a lot.

I am looking for the [Get Available Markets API](https://developer.spotify.com/documentation/web-api/reference/#/operations/get-available-markets). Is it missing?

In case it is missing, I created a PR for that https://github.com/JohnnyCrazy/SpotifyAPI-NET/pull/787

**Comment by JohnnyCrazy** [comment, marked as answer]
Thanks for the question and PR! I think spotify just silently adds new endpoints, at least I'm not aware of any news channel which one can subscribe to...

Will try to get a release out today in the evening

**Reply by esskar** [reply]
Nice. Thanks.

**Reply by esskar** [reply]
I know you are probably busy and it's the weekend, but any chance for pushing the changes as nuget?

**Reply by JohnnyCrazy** [reply]
Sorry for the late release, relocation took longer than expected! If there are any issues with the release, feel free to ping
